index.html

```diff
@@ -238,7 +333,7 @@ <h3 class="skillsTitle">Skills</h3>
             <h2 class="form-title">Contact me</h2>
             <p class="content">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
             <form action="https://formspree.io/f/mbjwowny" method="post">
-            <ul>
+            <ul class="inputs">
                 <li>
                     <label for="name"></label>
                     <input type="text" id="name" name="name" placeholder="Enter your name here ..." maxlength="30" required />
```

Comment:
- [ ] Kindly make your form and the description bigger to be like the Figma design. You can give it width with percentage %.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/35535130/155708711-0c96af8b-58e4-499d-836e-728a47113936.png) | ![image](https://user-images.githubusercontent.com/35535130/155708757-3ac949f1-50f7-4f63-8ed0-059e1f762b01.png) |